runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeResponse.kt

```diff
@@ -1,24 +1,44 @@
 package io.openruntimes.kotlin
 
-import io.javalin.plugin.json.JsonMapper
+import com.google.gson.GsonBuilder;
+import com.google.gson.Gson;
 
-class RuntimeResponse(
-    var data: Any = "",
-    var statusCode: Int = 200
-) {
-    fun json(data: Map<String, Any?>, statusCode: Int = 200): RuntimeResponse {
-        this.data = data
-        this.statusCode = statusCode
-        return this
+public class RuntimeResponse {
+    fun send(body: String, statusCode: Int, headers: MutableMap<String, String>): RuntimeOutput {
+        return RuntimeOutput(body, statusCode, headers);
     }
+    fun send(body: String, statusCode: Int): RuntimeOutput {
+        return this.send(body, statusCode, mutableMapOf<String, String>());
+    }
+    fun send(body: String): RuntimeOutput {
+        return this.send(body, 200, mutableMapOf<String, String>());
+    }
+
+    fun json(json: MutableMap<String, Any>, statusCode: Int, headers: MutableMap<String, String>): RuntimeOutput {
+        var gson = GsonBuilder().serializeNulls().create();
 
-    fun send(data: String, statusCode: Int = 200): RuntimeResponse {
-        this.data = data
-        this.statusCode = statusCode
-        return this
+        headers.put("content-type", "application/json");
```

Comment:
Kotlin maps should use indexed access, applies to all uses of `put` in the PR

```suggestion
        headers["content-type"] = "application/json"
```